MAINTAINING.md

```diff
@@ -34,17 +34,19 @@ the overall validity of the PR.
     Is the problem that the PR intends to solve clearly stated?
 2.  Is the solution presented in the PR as simple as possible to satisfy the
     requirements, but no simpler?
-3.  Is the solution well designed on a high level?
-4.  Do the concepts used by the PR make sense?
-5.  Does the PR break with existing concepts?
-6.  Is the structure of the PR itself valid?
-7.  Are there clear and adequate instructions on how to test the PR? \
+3.  Is the PR functionally atomic, does it address a single issue or should
+    it be split into multiple individual PRs?
```

Comment:
This is two sentences: "Is the PR functionally atomic? Does it address a single issue or should it be split into multiple individual PRs?"
Hope you don't mind me upholding my language. It's all we have left...

Reply:
Thanks, changed it.

> Hope you don't mind me upholding my language. It's all we have left...

Nah, I appreciate the grammar and spelling check here